### PR TITLE
Migration to sync team event properties

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Adds a migration to add event names and event properties to the new models
- Adds event names/props via delayed tasks
- Wraps it all with a try/except to avoid breaking in the future

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: posthog/migrations/0146_eventproperty_sync.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+def sync_team_event_names_and_properties(apps, schema_editor):
+    Team = apps.get_model("posthog", "Team")
+    for team in Team.objects.all():
+        try:
+            # importing locally in case the code ever moves
+            from posthog.tasks.sync_event_and_properties_definitions import sync_event_and_properties_definitions
+
+            sync_event_and_properties_definitions.delay(team.uuid)
```
Comment (macobo) [inline posthog/migrations/0146_eventproperty_sync.py:13]: Will celery workers have restarted at this point and be able to exec this?

**Review by paolodamico — APPROVED** [review]
lgtm!

File: posthog/migrations/0146_eventproperty_sync.py (hunk 1 above)
Comment (paolodamico) [reply posthog/migrations/0146_eventproperty_sync.py:13]: Hmmm yeah actually I think it's better to run this synchronously

Comment (mariusandra) [reply posthog/migrations/0146_eventproperty_sync.py]: Good point. We can't guarantee that, so removed.

**Pull request merged**